ci-operator/step-registry/openshift/e2e/test/balanced/openshift-e2e-test-balanced-commands.sh

```
@@ -0,0 +1,74 @@
+echo "Tests assigned to this shard: ${SHARD_TEST_COUNT} of ${TOTAL_TESTS}"
+echo "Assigned tests:"
+cat /tmp/shard-tests
+
+if [[ "${SHARD_TEST_COUNT}" -eq 0 ]]; then
+    echo "No tests assigned to this shard. Exiting successfully."
+    exit 0
+fi
+
+# Run only the assigned tests via --file (not using --shard-count/--shard-id)
+set -x
+openshift-tests run "${TEST_SUITE}" ${TEST_ARGS:-} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)